bump_pydantic/codemods/field.py

```diff
@@ -115,12 +116,16 @@ def leave_field_call(self, original_node: cst.Call, updated_node: cst.Call) -> c
             if m.matches(arg, m.Arg(keyword=m.Name())):
                 keyword = RENAMED_KEYWORDS.get(arg.keyword.value, arg.keyword.value)  # type: ignore
                 value = arg.value
-                # The `allow_mutation` keyword argument is a special case. It's the negative of `frozen`.
-                if arg.keyword and arg.keyword.value == "allow_mutation":
-                    if m.matches(arg.value, m.Name(value="False")):
-                        value = cst.Name("True")
-                    elif m.matches(arg.value, m.Name(value="True")):
-                        value = cst.Name("False")
+                if arg.keyword:
+                    if arg.keyword.value == "allow_mutation":
+                        # The `allow_mutation` keyword is the negative of `frozen`.
+                        if m.matches(arg.value, m.Name(value="False")):
+                            value = cst.Name("True")
+                        elif m.matches(arg.value, m.Name(value="True")):
+                            value = cst.Name("False")
+                    if arg.keyword.value == "example":
+                        # The example keyword is now a list, `examples`.
+                        value = cst.List([arg.value])
```

Comment:
@Gamrix 
The following worked for me

value = cst.List([cst.Element(arg.value)])